lib/Settings/Navigation.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:cboard_mobile/shared/app-bar.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
What does the `cupertino` package do?

Reply:
it's default package for Divider() widget

Reply:
Ohh, okay - sounds good!